src/client/signatures.rs

```diff
@@ -112,13 +112,7 @@ mod tests {
             if should_fail {
                 assert!(verifier.verify(&collection).is_err())
             } else {
-                match verifier.verify(&collection) {
-                    Err(err) => {
-                        println!("{:?}", err);
-                        assert!(false)
-                    }
-                    Ok(_) => println!("success"),
-                }
+                assert!(verifier.verify(&collection).is_ok());
```

Comment:
This won't print the error, since the argument to `assert!()` is just a bool in this version, but I guess it's ok anyway – this isn't likely to fail often, and it's most important we are testing the right thing.